applications/RomApplication/custom_utilities/rom_residuals_utility.h

```diff
@@ -138,17 +140,183 @@ namespace Kratos
         return MatrixResiduals;
         }
 
+        Matrix GetProjectedResidualsOntoPsi()
+        {
+            // Getting the number of elements and conditions from the model
+            const int nelements = static_cast<int>(mpModelPart.Elements().size());
+            const int nconditions = static_cast<int>(mpModelPart.Conditions().size());
+
+            const auto& CurrentProcessInfo = mpModelPart.GetProcessInfo();
```

Comment:
```suggestion
            const auto& r_current_process_info = mpModelPart.GetProcessInfo();
```
See our [style guide](https://github.com/KratosMultiphysics/Kratos/wiki/Style-Guide).